Fulfillment/Troubleshooting/01_HandleErrorStatus.md

```diff
@@ -0,0 +1,91 @@
+[!!Export a dispatch note](../Operation/01_ManageDispatchNotes.md#export-a-dispatch-note) 
+[!!Workflows](../../ActindoWorkFlow/Overview/01_General.md) 
+
+# Handle an error status
+
+The *Fulfillment* module manages the communication between the *Actindo Core1 Platform* and the fulfillment partner(s). Occasionally, a dispatch note or a shipment may display an error.
+
+An error status may be set for many different reasons and strongly depends on the connection and the fulfiller's own internal processes. 
+
+A dispatch note usually displays an error status when the data export from the Core1 to the partner's system has failed. A shipment, on the other hand, may display an error status when the fulfiller has reported an error, for instance because the shipment could not be arranged, but also when no feedback from fulfiller has been received. The errors are notified via the ![Notifications](../../Assets/Icons/Notifications.png) icon (Notifications) on the upper right corner as they occur.
+
+In any case, the corresponding workflow stops at the point where the error occurs. The *Actindo Core1 Platform* logging system provides detailed information on any error, which allows to identify the cause and implement the appropriate solution.
+
+#### Error Description
+
+The *Status* or the *Shipment status* columns in the list of dispatch notes display an error.
+
+Follow the instructions below to identify the cause of the error and proceed accordingly.
+
+#### Prerequisites
+
+No prerequisites to fulfill.
+
+#### Procedure
+
+*Fulfillment > Dispatch notes > Tab LIST*
+
+![Error status](../../Assets/Screenshots/Fulfillment/DispatchNotes/DispatchNotesErrorStatus.png "[Error status]")
+
+1. Click the dispatch note displaying an error in the *Status* or the *Shipment status* columns.  
+    The *Dispatch note "Dispatch note ID"* view is displayed. 
+
+2. Click the *Logs* sub-tab.  
+    The list of intents related to the selected dispatch note are displayed.
+
+    ![Logs](../../Assets/Screenshots/Fulfillment/DispatchNotes/DispatchNoteLogsError.png "[Logs]")
+
+3. Click the desired intent from the list to access further information.  
+    The list of messages is displayed.  
+
+    ![Messages](../../Assets/Screenshots/Fulfillment/DispatchNotes/DispatchNoteLogsMessages.png "[Messages]")
+
+4. If necessary, click the relevant message for further information.  
+    The *Detail of log message "Log message title"* is displayed.
+ 
```

Comment:
Mhh. Kommt jetzt im Anschluss noch ne andere Möglichkeit, die Infos zu bekommen? Dann vielleicht nicht in einer sondern in zwei Procedures beschreiben?!